### ISSUE TITLE
Install failed - could not install package ... into a project that targets '.NETFramework,Version=v4.0'

### ISSUE THREAD

**Issue by TomasJuocepis** [vkhorikov/CSharpFunctionalExtensions#24]
Trying to install the package into .NET 4.0 project, but it fails. Is .NET 4.0 not supported or is something else going wrong?

Error:

```
Install failed. Rolling back...
Install-Package : Could not install package 'CSharpFunctionalExtensions 1.7.1'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.0', but the package does not contain any assembly references or content files that are
 compatible with that framework. For more information, contact the package author.
At line:1 char:16
+ Install-Package <<<<  CSharpFunctionalExtensions
    + CategoryInfo          : NotSpecified: (:) [Install-Package], InvalidOperationException
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PowerShell.Commands.InstallPackageCommand
 
PM> 
```

**Comment by vkhorikov** [comment]
Hi Tomas, sorry for the late reply. Yes, the library has compatibility issues with .NET 4.0, I'll try to fix them shortly. It the meanwhile, you might be able to use this forked version: CSharpFunctionalExtensions.Net40

**Comment by mjharwood** [comment]
Support for .net 4.0 is available as of v1.8.1
